cicd/jenkins/openshift-jenkins.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: ASSEMBLY
+[id="openshift-jenkins_{context}"]
+= {product-title} Jenkins
+include::_attributes/common-attributes.adoc[]
+:context: openshift-jenkins
+
+toc::[]
+
+{product-title} 4.11 moves the `OpenShift Jenkins` and `OpenShift Agent Base` images to the `ocp-tools-4` repository at `registry.redhat.io` so that Red Hat can produce and update the images outside the {product-title} lifecycle. Previously, these images were in the {product-title} install payload and the `openshift4` repository at `registry.redhat.io`.
+
+{product-title} 4.10 deprecated the `OpenShift Jenkins Maven` and `NodeJS Agent` images. {product-title} 4.11 removes these images from its payload. Red Hat no longer produces these images, and they are not available from the `ocp-tools-4` repository at `registry.redhat.io`. Red Hat maintains the 4.10 and earlier versions of these images for any bug fixes or security CVEs of note, in accordance the link::https://access.redhat.com/support/policy/updates/openshift[{product-title} lifecycle policy].
```

Comment:
well, maybe this first statemen with 

and they are not available from the `ocp-tools-4` repository at `registry.redhat.io`

is still a  bit confusing/misleading ..... the maven and nodejs image for 4.10 and earlier are still available at `ocp-tools-4`, but there will be no versions of those images at 4.11 and later.

maybe the second statement fixes it, but I'm not 100% sure.

What do you think @rolfedh @coreydaley @akram @rgormley  - do we need a follow up PR to fix the possible lack of clarify here.

Reply:
@gabemontero:  I'm happy to make any improvements you suggest. 
- The complete sentence reads: "Red Hat maintains the 4.10 and earlier versions of these images for any bug fixes or security CVEs of note, in accordance the link::https://access.redhat.com/support/policy/updates/openshift[{product-title} lifecycle policy]."
- Additionally, the "Relocation of OpenShift Jenkins images" topic mentions: "The "OpenShift Jenkins Maven" and "NodeJS Agent" images for {product-title} were deprecated in 4.10, and are removed from the {product-title} install payload in 4.11. They do not have alternatives defined in the `ocp-tools-4` repository. However, you can work around this by using the sidecar pattern described in the "Jenkins agent" topic mentioned in the following "Additional resources" section."

Reply:
ok I'm probably being overly worrisome .... unless I've stirred up angst with @coreydaley @akram @rgormley let's leave things as is and see if customers report confusion.